packages/bot-web-ui/.eslintrc.js

```diff
@@ -17,4 +17,51 @@ module.exports = {
             webpack: { config: webpackConfig({}) },
         },
     },
-};
\ No newline at end of file
+    plugins: ['simple-import-sort'],
+    rules: {
+        'simple-import-sort/imports': 'warn',
+        'simple-import-sort/exports': 'warn',
+    },
+    overrides: [
+        {
+            files: ['**/*.js', '**/*.ts', '**/*.tsx', '**/*.jsx'],
+            rules: {
+                'simple-import-sort/imports': [
+                    'warn',
+                    {
+                        groups: [
+                            [
+                                'public-path',
+                                // `react` first, then packages starting with a character
+                                '^react$',
+                                '^[a-z]',
+                                // Packages starting with `@`
+                                '^@',
+                                // Packages starting with `~`
+                                '^~',
+                                '^Components',
+                                '^Constants',
+                                '^Utils',
+                                '^Types',
+                                '^Stores',
+                                // Imports starting with `../`
+                                '^\\.\\.(?!/?$)',
+                                '^\\.\\./?$',
+                                // Imports starting with `./`
+                                '^\\./(?=.*/)(?!/?$)',
+                                '^\\.(?!/?$)',
+                                '^\\./?$',
+                                // Style imports
+                                '^.+\\.s?css$',
+                                // Side effect imports
+                                '^\\u0000',
```

Comment:
A side effect import is an import that only imports a module for its side effects, without importing any values (i.e. where ImportDeclaration. specifiers is an empty array).